.github/CHANGELOG.md

```diff
@@ -83,6 +57,245 @@
+      qml.Hadamard(wires=0)
+      qml.PauliZ(wires=1)
+      qml.Hadamard(wires=0)
+      qml.T(wires=0)
+      qml.CZ(wires=[0, 1])
+      qml.CZ(wires=[1, 0])
+      return qml.expval(qml.PauliX(wires=0))
+  ```
+
+  ```pycon
+  >>> dev = qml.device('default.qubit', wires=2)
+  >>> qnode = qml.QNode(circuit, dev)
+  >>> print(qml.draw(qnode)())
+   0: ──H──H──T──╭C──╭Z──┤ ⟨X⟩
+   1: ──Z────────╰Z──╰C──┤
+  >>> optimized_circuit = qml.transforms.cancel_inverses(circuit)
+  >>> optimized_qnode = qml.QNode(optimized_circuit, dev)
+  >>> print(qml.draw(optimized_qnode)())
+   0: ──T──┤ ⟨X⟩
+   1: ──Z──┤
+  ```
+
+  The `merge_rotations` transform combines adjacent rotation gates of
+  the same type into a single gate, including controlled rotations.
+
+  ```python
+  def circuit(x, y, z):
+      qml.RX(x, wires=0)
+      qml.RX(x, wires=0)
+      qml.Rot(x, y, z, wires=1)
+      qml.Rot(y, z, x, wires=1)
+      qml.CRY(y, wires=[0, 1])
+      qml.CRY(y + z, wires=[0, 1])
+      return qml.expval(qml.PauliX(wires=0))
+  ```
+
+  ```pycon
+  >>> qnode = qml.QNode(circuit, dev)
+  >>> print(qml.draw(qnode)(0.1, 0.2, 0.3))
+   0: ──RX(0.1)─────────────RX(0.1)─────────────╭C────────╭C────────┤ ⟨X⟩
+   1: ──Rot(0.1, 0.2, 0.3)──Rot(0.2, 0.3, 0.1)──╰RY(0.2)──╰RY(0.5)──┤
+  >>> optimized_circuit = qml.transforms.merge_rotations()(circuit)
+  >>> optimized_qnode = qml.QNode(optimized_circuit, dev)
+  >>> print(qml.draw(optimized_qnode)(0.1, 0.2, 0.3))
+   0: ──RX(0.2)───────────────────╭C────────┤ ⟨X⟩
+   1: ──Rot(0.409, 0.485, 0.306)──╰RY(0.7)──┤
+  ```
+
+* A new quantum function transform has been added to perform full fusion of
+  arbitrary-length sequences of single-qubit gates.
+  [(#1458)](https://github.com/PennyLaneAI/pennylane/pull/1458)
+
+  The `single_qubit_fusion` transform acts on all sequences of
+  single-qubit operations in a quantum function, and converts each
+  sequence to a single `Rot` gate. For example given the circuit:
+
+  ```python
+  def circuit(x, y, z):
+      qml.Hadamard(wires=0)
+      qml.PauliZ(wires=1)
+      qml.RX(x, wires=0)
+      qml.RY(y, wires=1)
+      qml.CZ(wires=[1, 0])
+      qml.T(wires=0)
+      qml.SX(wires=0)
+      qml.Rot(x, y, z, wires=1)
+      qml.Rot(z, y, x, wires=1)
+      return qml.expval(qml.PauliX(wires=0))
+  ```
+
+  ```pycon
+  >>> optimized_circuit = qml.transforms.single_qubit_fusion()(circuit)
+  >>> dev = qml.device('default.qubit', wires=2)
+  >>> qnode = qml.QNode(optimized_circuit, dev)
+  >>> print(qml.draw(qnode)(0.1, 0.2, 0.3))
+   0: ──Rot(3.24, 1.57, 0)──╭Z──Rot(2.36, 1.57, -1.57)────┤ ⟨X⟩
+   1: ──Rot(3.14, 0.2, 0)───╰C──Rot(0.406, 0.382, 0.406)──┤
+  ```
+
+<h4>Improved Hamiltonian simulations</h4>
+
+* Added a sparse Hamiltonian observable and the functionality to support computing its expectation
+  value. [(#1398)](https://github.com/PennyLaneAI/pennylane/pull/1398)
+
+  For example, the following QNode returns the expectation value of a sparse Hamiltonian:
```

Comment:
I might be careful with this one:

- Only `default.qubit` is supported at the moment. However the entry makes it sound like it leads to faster execution in general.
- Only expvals are support, I believe(?)

Finally, I think the plan is to replace it with `qml.expval(qml.Hamiltonian(...))`, and simply use the fact that `qml.Hamiltonian` is _already_ a sparse representation of the Hamiltonian! So not sure how much we want to publicize this, since this change will likely be in the next release.

@mariaschuld, any thoughts here?

Reply:
Perhaps it should be sufficient just to move this category 'lower' down the list of new features

Reply:
Sounds good! Moved it lower and added a note on `default.qubit`.